### PR TITLE
[FIX] mail: how to choose allowed "From" addresses

### PR DESCRIPTION
**Pull request by Yajo**
- Document changes from https://github.com/odoo/odoo/pull/69218.
- Explain that Office365 is now usable.

@Tecnativa TT29842

### CODE REVIEW

**Comment by std-odoo** [comment]
@Yajo Good explanation :)

Note that this feature will be removed in master
(see https://www.odoo.com/web#id=2367946&action=333&active_id=965&model=project.task&view_type=form&cids=1&menu_id=4720 )

And a "smarter" behavior will be implemented

File: content/applications/productivity/discuss/email_servers.rst [hunk 1]
```diff
@@ -84,6 +85,48 @@ that is a CNAME with the value "odoo._domainkey.odoo.com".
...
+a message, Odoo will try to redistribute that same email to other subscribers
+in the thread. But if the domain *customer.example.com* forbids that kind of
+usage for security (kudos for that), the Odoo's redistributed email would get
+rejected by some recipients' mail servers.
+
+To avoid those kind of problems, you should make sure all emails use a "From"
+address from your authorized domain.
+
+If your MTA supports `SRS (Sender Rewriting Scheme)
+<https://en.wikipedia.org/wiki/Sender_Rewriting_Scheme>`__, you can enable it
```
Comment (AntoineVDV) [inline content/applications/productivity/discuss/email_servers.rst:104]:
```suggestion
<https://en.wikipedia.org/wiki/Sender_Rewriting_Scheme>`_, you can enable it
```
See https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#use-single-underscore-suffixes-for-hyperlink-references

File: content/applications/productivity/discuss/email_servers.rst [hunk 2]
```diff
@@ -84,6 +85,48 @@ that is a CNAME with the value "odoo._domainkey.odoo.com".
...
+to handle these situations. However, that is more complex and requires more
+technical knowledge that is not meant to be covered by this documentation.
+
+Instead, you can also configure Odoo to do something similar by itself:
+
+#.  Set your domain name in the General Settings.
+
+    .. image:: media/alias_domain.png
+      :align: center
```
Comment (AntoineVDV) [inline content/applications/productivity/discuss/email_servers.rst:113]:
```suggestion
       :align: center
```
Nitpicking: https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#be-consistent-with-indentation

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Hello, please find below the few (nitpicking) comments that concern the technical aspect of your PR. Your line size limit of 80 chars is a bit overkill IMO ([100 is recommended for this repo](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character)) but it's fine as is. Pretty good RST apart from that ;)

**Comment by Yajo** [comment]
> Your line size limit of 80 chars is a bit overkill IMO ([100 is recommended for this repo](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character)) but it's fine as is.

You should consider adopting https://editorconfig.org/ (even better if combined with https://pre-commit.com) if you really care about this.

> Pretty good RST apart from that ;)

Thanks... RST is though. 😅

**Comment by AntoineVDV** [comment]
> You should consider adopting https://editorconfig.org/ (even better if combined with https://pre-commit.com) if you really care about this.
@Yajo We'll soon integrate this repository with Odoo's runbot ;)

**Review by AntoineVDV — APPROVED** [review]

**Comment by AntoineVDV** [comment]
Thank you for your contribution ;)

**Comment by Yajo** [comment]
I don't see this change deployed, nor migrated to newer branches... is there something else to do?

**Comment by AntoineVDV** [comment]
> I don't see this change deployed, nor migrated to newer branches... is there something else to do?

Hello, this change had indeed been deployed. See [here](https://www.odoo.com/documentation/14.0/applications/productivity/discuss/advanced/email_servers.html#choose-allowed-from-email-addresses) for example.

**Comment by Yajo** [comment]
🤦🏼‍♂️ I don't know how I searched then. Thanks, sorry.

**Pull request merged**